packages/@aws-cdk/aws-amplify/test/app.test.ts

```diff
@@ -442,3 +442,19 @@ test('with custom headers', () => {
     },
   });
 });
+
+test('create an amplify app with platform ', () => {
```

Comment:
This single test doesn't cover all the updates. Please increase the test coverage on your changes.

Reply:
@TheRealAmazonKendra Hi, Amplify PM here, I think this test might cover the change made to the `platform` updates applied at the Amplify App level.  The `framework` property tests are done in the `branch.test.ts`.